Update Universal Profile guides (use LSP20) (#483)
lukso-network/docs: docs/guides/universal-profile/interact-with-contracts.md

The first step is to create instances of our Universal Profile, Key Manager contracts and the Target Contract.

- Create an Universal Profile contract instance from `universalProfileAddress`.

> - Get the `owner()` of the Universal Profile.

Reviewer: great 👌

## Step 2 - Encode the calldatas

> This is the easy part, we need to create a calldata that will be executed on the Target Contract.

Reviewer: ```suggestion
We need to create a calldata that will be executed on the Target Contract.
```

### Send the execute calldata

> The final step is to pass the encoded calldata to the Universal Profile. Since we are calling from a UP's controller address (with proper [**permissions**](../../standards/universal-profile/lsp6-key-manager.md#permissions)), the Key Manager will authorize the transaction.

Reviewer: + add link to to LSP6 page to explain what a controller is.

```suggestion
The final step is to pass the encoded calldata to the Universal Profile. Since we are calling from an EOA that is a controller on the UP, the Key Manager will authorize the transaction.
```
